app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -312,8 +331,12 @@ class TabTrayView(
         components.analytics.metrics.track(eventToSend)
     }
 
-    fun expand() {
-        behavior.state = BottomSheetBehavior.STATE_EXPANDED
+    fun accomodateTabs() {
```

Comment:
Let's add a kDoc for this. I want to encourage clearer comments for functionality overall.

```
/**
 * Updates the bottom sheet visibility based on the number tabs or screen orientation.
 * Show the bottom sheet fully expanded if it is in landscape mode or the number of 
 * tabs are greater or equal to the expand size limit.
 */
```

Reply:
I don't see the intent of this as to update the visibility (as in visible / invisible) but rather the height of the bottom sheet.
Could I change the comment to
```
/**
 * Updates the bottom sheet height based on the number tabs or screen orientation.
 * Show the bottom sheet fully expanded if it is in landscape mode or the number of 
 * tabs are greater or equal to the expand size limit.
 */
```
?

Reply:
Sounds good